sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -31140,7 +31140,7 @@
                     "strictDateValidation": false,
                     "multiple": false,
                     "unique": false,
-                    "onlyAvailableItems": false
```

Comment:
There is a radio at line 31301.

Reply:
That line should be connected to the change at line 31298.